ee/debug/checkups/power_windows.go

```diff
@@ -40,15 +41,43 @@ func (p *powerCheckup) Run(ctx context.Context, extraWriter io.Writer) error {
 	}
 	defer sprHandle.Close()
 
-	if _, err := io.Copy(extraWriter, sprHandle); err != nil {
+	extraZip := zip.NewWriter(extraWriter)
+	defer extraZip.Close()
+
+	zippedPowerReport, err := extraZip.Create("power.html")
+	if err != nil {
+		return fmt.Errorf("creating power report zip file: %w", err)
+	}
+
+	if _, err := io.Copy(zippedPowerReport, sprHandle); err != nil {
 		return fmt.Errorf("copying system power report: %w", err)
 	}
 
+	powerCfgSleepStatesCmd, err := allowedcmd.Powercfg(ctx, "/availablesleepstates")
+	if err != nil {
+		return fmt.Errorf("creating powercfg sleep states command: %w", err)
+	}
+
+	hideWindow(powerCfgSleepStatesCmd)
+	availableSleepStatesOutput, err := powerCfgSleepStatesCmd.CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("running powercfg.exe for sleep states: error %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("running powercfg.exe for sleep states: error %w, output %s", err, string(availableSleepStatesOutput))
```

IIRC on Windows we don't really get anything useful in the error and need to include stdout to see what the actual issue is

Reply:
ahh okay, thank you!